File: Python3/2351_first_letter_to_appear_twice.py
# https://leetcode.cn/problems/first-letter-to-appear-twice/

class Solution:
    def repeatedCharacter(self, s):
        """
        Inputs:
            s: str
        Outputs:
            res: str
        """
        cnt = [0 for i in range(26)]
        for ch in s:
            if cnt[ord(ch) - ord("a")]:
                return ch
            cnt[ord(ch) - ord("a")] = 1
        return "-"

File: Cpp/2351_first_letter_to_appear_twice.cpp
// https://leetcode.cn/problems/first-letter-to-appear-twice/

class Solution {
public:
    char repeatedCharacter(string s) {
        vector<int> cnt(26);
        for (char ch: s) {
            if (cnt[ch-'a']) {
                return ch;
            }
            cnt[ch-'a'] = 1;
        }
        return '-';
    }
};